.circleci/config.yml

```diff
@@ -109,95 +117,101 @@ jobs:
   verify-dags-up-to-date:
     docker: *docker
     steps:
-    - checkout
-    - *build
-    - run: 
-        name: Generate DAGs
-        command: PATH="venv/bin:$PATH" script/generate_airflow_dags
-    - run:
-        name: Verify that DAGs were correctly generated and are up-to-date
-        command: git diff --exit-code
+      - checkout
+      - *build
+      - run:
+          name: Generate DAGs
+          command: PATH="venv/bin:$PATH" script/generate_airflow_dags
+      - run:
+          name: Verify that DAGs were correctly generated and are up-to-date
+          command: git diff --exit-code
   validate-docs:
     docker: *docker
     steps:
-    - checkout
-    - *build
-    - run:
-        name: Validate doc examples
-        command: PATH="venv/bin:$PATH" script/validate_docs
+      - checkout
+      - *build
+      - run:
+          name: Validate doc examples
+          command: PATH="venv/bin:$PATH" script/validate_docs
   docs:
     docker: *docker
     steps:
-    - checkout
-    - &skip_forked_pr
-      run:
-        name: Early return if this build is from a forked PR
-        command: |
-          if [ -n "$CIRCLE_PR_NUMBER" ]; then
-            echo "Cannot pass creds to forked PRs, so marking this step successful"
-            circleci step halt
-          fi
-    - *build
-    - run:
-        name: Install dependencies
-        command: pip install mkdocs markdown-include
-    - add_ssh_keys:
-        fingerprints:
-          "ab:b5:f7:55:92:0a:72:c4:63:0e:57:be:cd:66:32:53"
-    - run:
-        name: Build and deploy docs
-        command: |
-          PATH="venv/bin:$PATH" script/generate_docs --output_dir=generated_docs/
-          cd generated_docs/
-          mkdocs gh-deploy -m "[ci skip] Deployed {sha} with MkDocs version: {version}"
+      - checkout
+      - &skip_forked_pr
+        run:
+          name: Early return if this build is from a forked PR
+          command: |
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+            echo "Cannot pass creds to forked PRs,
+              so marking this step successful"
+              circleci step halt
+            fi
+      - *build
+      - run:
+          name: Install dependencies
+          command: pip install mkdocs markdown-include
+      - add_ssh_keys:
+          fingerprints: "ab:b5:f7:55:92:0a:72:c4:63:0e:57:be:cd:66:32:53"
+      - run:
+          name: Build and deploy docs
+          command: |
+           PATH="venv/bin:$PATH" script/generate_docs
+            --output_dir=generated_docs/
```

Comment:
This is going to break the script because it doesn't have a proper line break. Is it not possible to keep this on one line? Otherwise I suggest this:

```bash
PATH="venv/bin:$PATH" script/generate_docs \
     --output_dir=generated_docs/
```

Reply:
I had to input line breaks to fix another error, apparently lines can only be 80 characters long. If this is a better way to introduce line breaks, I can do that

Reply:
Yes, please introduce the backslash at the end of the first line, or bash will treat the second line as it's own command. This will lead to subtle bugs.